### ISSUE TITLE
VOIP not functioning

### ISSUE THREAD

**Issue by LongStryder** [StrangeLoopGames/EcoIssues#20131]
I was asked to create an issue here in response to my support request, on what I believed to be a common issue. Recently, when seeing some content creators stream Eco, I saw that it was a fairly common occurrence for their VOIP to spontaneously combust and stop functioning, prompting them to restart the game, and voila - their problems are gone. Today marks my fifth day of playing the game without being able to hear or speak to other players. I keep finding myself in awkward positions where it is clear someone is speaking to me and all I can do is stand there and stare at them, and hope they notice me typing to them in chat. Upon researching online, I was only able to find tips about setting my Windows default microphone, which I had already done, and restarting the game, which hasn't worked for me. I even tried updating the software on my headset, which is a Logitech G933, and I tried verifying my Eco steam install files, none of which have done anything to help the issue. My voice chat is turned on in game, and I've even tried turning it on and back off, although I haven't turned it off, restarted, and then turned it back on, which I suppose I could try. I've just been stuck in a perpetual loop of not being able to hear anything but game sounds, or having to listen to the conversations going on through the twitch livestream when the server owner is streaming the game. I hope that my log files will show something that can help pinpoint what is going on. 

[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/5857152/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/5857153/Player-prev.log)

**Comment by redwyre** [comment]
Hi @LongStryder 
It looks like the error you are getting is undocumented, I'm requesting info on what it is.
Are you running windows 10?

**Comment by IcedForge** [comment]
Adding the initial connection error code piece for search functionality.

```
[10:19:46.865] <BeginGetConnectorHandle>b__0: vx_req_connector_create_t failed: VivoxUnity.VivoxApiException:  (11175)
  at VivoxUnity.VxClient.EndIssueRequest (System.IAsyncResult result) [0x00000] in <00000000000000000000000000000000>:0 
  at VivoxUnity.Client+<>c__DisplayClass12_0.<BeginGetConnectorHandle>b__0 (System.IAsyncResult ar) [0x00000] in <00000000000000000000000000000000>:0 
  at VivoxUnity.VxClient+HandleEventMessage.Invoke (vx_evt_base_t eventMessage) [0x00000] in <00000000000000000000000000000000>:0 
  at VivoxUnity.AsyncResult`1[T].SetComplete (T result) [0x00000] in <00000000000000000000000000000000>:0 
  at VivoxUnity.VxClient.InstanceOnMainLoopRun (System.Boolean& didWork) [0x00000] in <00000000000000000000000000000000>:0 
  at MessagePump.RunOnce () [0x00000] in <00000000000000000000000000000000>:0 
  at MessagePump.RunUntil (LoopDone done) [0x00000] in <00000000000000000000000000000000>:0 
  at VoiceManager.Update () [0x00000] in <00000000000000000000000000000000>:0 
EcoEngine.Logging.LogManager:LogFormat(LogType, Object, String, Object[])
UnityEngine.Logger:Log(LogType, Object)
UnityEngine.Debug:LogError(Object)
VivoxUnity.VivoxDebug:DebugMessage(Object, vx_log_level)
VivoxUnity.<>c__DisplayClass12_0:<BeginGetConnectorHandle>b__0(IAsyncResult)
VivoxUnity.HandleEventMessage:Invoke(vx_evt_base_t)
VivoxUnity.AsyncResult`1:SetComplete(T)
VivoxUnity.VxClient:InstanceOnMainLoopRun(Boolean&)
MessagePump:RunOnce()
MessagePump:RunUntil(LoopDone)
VoiceManager:Update()
```

**Comment by LongStryder** [comment]
Yes, I am running Windows 10.

**Comment by redwyre** [comment]
waiting on a response from vivox

**Comment by Sjanoki** [comment]
I'm having the exact same issue, but it looks like I'm at a dead end
